docs/machine-learning/tutorials/image-classification.md

```diff
@@ -0,0 +1,400 @@
+{
+
+}
+```
+
+### Load the data
+
+Data in ML.NET is represented as an [IDataView class](xref:Microsoft.Data.DataView.IDataView). `IDataView` is a flexible, efficient way of describing tabular data (numeric and text). Data can be loaded from a text file or in real time (for example, SQL database or log files) to an `IDataView` object.
+
+Load the data using the `MLContext.Data.ReadFromTextFile` wrapper. Add the following code as the next line in the BuildAndTrainModel method:
+
+[!code-csharp[LoadData](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#LoadData "Load the data")]
+
+### Extract Features and transform the data
+
+Pre-processing and cleaning data are important tasks that occur before a dataset is used effectively for machine learning.  Using data without these modeling tasks can produce misleading results.
+
+ML.NET's transform pipelines compose a custom `transforms`set that is applied to your data before training or testing. The transforms' primary purpose is data [featurization](../resources/glossary.md#feature-engineering). Machine learning algorithms understand [featurized](../resources/glossary.md#feature) data, and when dealing with deep neural networks you must adapt the images to the format expected by the network. That format is a [numeric vector](../resources/glossary.md#numerical-feature-vector).
+
+When the model is trained and evaluated, by default, the values in the **Label** column are considered as correct values to be predicted. As you are using a pre-trained model you need to map fields to the new model. To do that, use the `MLContext.Transforms.Conversion.MapValueToKey`.  Name this `estimator` as you will then append the trainer to it. Add the next line of code:
+
+[!code-csharp[MapValueToKey1](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#MapValueToKey1)]
+
+Your image processing estimator uses pre-trained DNN featurizers for feature extraction. You use several image transforms to get the image data in the expected form:
+
+1. The `LoadImages`transform images are loaded in memory as a Bitmap type. 
+2. The `Resize` transform resizes the images as the pre-trained model has a defined input image width and height.
+3. The `ImagePixelExtractingEstimator` transform extracts the pixels from the input images and converts them into a numeric vector.
+
+Add these image transforms as the next lines of code:
+
+[!code-csharp[ImageTransforms](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ImageTransforms)]
+
+TensorFlow is a popular deep learning and machine learning toolkit that enables training deep neural networks (and general numeric computations), and is implemented as a transformer in ML.NET.  It's used to reuse the Inception model. The `TensorFlowTransform` extracts specified outputs (the model's image features `softmax2_pre_activation`) , and scores a dataset using a pre-trained Tensorflow model.
```

Comment:
I would move the first sentence (explaining what TensorFlow is further up in the tutorial). Then, in this section I'd mention that TensorFlow is implemented as a transformer in ML.NET.

Change last Tensorflow to TensorFlow